### PR TITLE
RIA-4928

### PR DESCRIPTION
**Pull request by srinivasuluputcha**
Bundling: Clarifying questions document template set up

### Bundling: Clarifying questions document template set up ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities;
+
+import java.util.List;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field.Document;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field.IdValue;
+
```
Comment (bradsorour) [inline src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java:8]: Could possibly introduce lombok `@Getter` and `@Setter` annotations, but happy to leave as is.

Comment (srinivasuluputcha) [reply src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java:8]: @bradsorour  good catch, i will add it

**Review by bradsorour — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java (hunk 1 above)
Comment (srinivasuluputcha) [reply src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java:10]: This is now added.

**Review by bradsorour — APPROVED** [review]
LGTM

**Pull request merged**